pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -835,6 +835,76 @@ where
         )
     }
 
+    pub fn gr_reservation_reply_commit_per_kb(n: u32) -> Result<Exec<T>, &'static str> {
+        let payload_offset = 1;
+        let payload_len = n as i32 * 1024;
+        let r = 1;
+
+        let rid_value_offset = 1;
+
+        let rid_values = (0..r)
+            .flat_map(|i| {
+                let mut bytes = [0; 32 + 16];
+                bytes[..32].copy_from_slice(ReservationId::from(i as u64).as_ref());
+                bytes.to_vec()
+            })
+            .collect::<Vec<_>>();
+
+        let err_mid_offset = rid_value_offset + rid_values.len() as u32;
+        let err_len_ptrs = Self::err_len_ptrs(1, err_mid_offset);
+
+        let code = WasmModule::<T>::from(ModuleDefinition {
+            memory: Some(ImportedMemory::max::<T>()),
+            imported_functions: vec![SysCallName::ReservationReply],
+            data_segments: vec![DataSegment {
+                offset: rid_value_offset,
+                value: rid_values,
+            }],
+            handle_body: Some(body::plain(vec![
+                // rid_value ptr
+                Instruction::I32Const(rid_value_offset as i32),
+                // payload ptr
+                Instruction::I32Const(payload_offset),
+                // payload len
+                Instruction::I32Const(payload_len),
+                // delay
+                Instruction::I32Const(10),
+                // err_mid ptr
+                Instruction::I32Const(err_mid_offset as i32),
```

Comment:
Possibly `i32.const` must be used instead of counter in other benches too

Reply:
We can't use `body::plain()` in other benches. It won't be accurate. `ERR_LEN_SIZE` is 4 bytes and it can't exceed even 1 gear page.